diff --git a/linux/org.frescobaldi.Frescobaldi.metainfo.xml.in b/linux/org.frescobaldi.Frescobaldi.metainfo.xml.in
index a80463b2..ef0b631f 100644
--- a/linux/org.frescobaldi.Frescobaldi.metainfo.xml.in
+++ b/linux/org.frescobaldi.Frescobaldi.metainfo.xml.in
@@ -67,135 +67,12 @@
   <translation type="gettext">frescobaldi</translation>
 
   <releases>
-    <release version="3.1.3" date="2020-12-26">
-      <description>
-        <p>Notes:</p>
-        <ul>
-          <li>This release contains several bug fixes and improvements. It is probably the last release in the 3.1 version range</li>
-          <li>Full history and credits for this release are available at https://github.com/frescobaldi/frescobaldi/compare/v3.1.2...v3.1.3</li>
-        </ul>
-        <p>Features:</p>
-        <ul>
-          <li>a Clear Music View button and menu action were added</li>
-        </ul>
-        <p>Bug fixes:</p>
-        <ul>
-          <li>fixed "NameError: name 'imp' is not defined" when importing</li>
-          <li>fixed search of generated files on macOS for some Unicode file names</li>
-          <li>fixed selection of Python on Mac OS:</li>
-          <ul>
-            <li>select the system Python 2 or 3 according to LilyPond's version</li>
-            <li>support MacPorts' LilyPond tools</li>
-            <li>add option to allow forcing the use of the tools' #! lines (useful for self-compiled or other nonstandard LilyPond installations)</li>
-          </ul>
-          <li>fixed "AttributeError: 'PreviewJob' object has no attribute 'lilypond_version'"</li>
-          <li>fixed Ghostscript error on Mac with MacPorts' LilyPond 2.21.x</li>
-        </ul>
-        <p>Translations:</p>
-        <ul>
-          <li>Updated translations: Dutch, Czech, Russian</li>
-        </ul>
-      </description>
-    </release>
-
-    <release version="3.1.2" date="2020-04-13">
-      <description>
-        <p>Bug fixes:</p>
-        <ul>
-          <li>Fixed the error "AttributeError: 'NoneType' object has no attribute 'cursor'"</li>
-          <li>Fixed the "Two Pages (first page right)" mode</li>
-          <li>Fixed pinch gesture zooming in Music View on Mac OS X</li>
-          <li>Fixed printing the music score on Mac OS X</li>
-          <li>Fixed cursor navigation keyboard shortcuts on Mac OS X; new shortcuts:</li>
-          <ul>
-            <li>next document: ctrl+tab</li>
-            <li>previous document: ctrl+shift+tab</li>
-            <li>start of line: cmd+left</li>
-            <li>end of line: cmd+right</li>
-          </ul>
-          <li>Fixed the display of filenames in document tabs on Mac OS X</li>
-          <li>Fixed the global menu with no windows open on Mac OS X</li>
-          <li>Fixed the error with convert-ly on in the Mac app bundle</li>
-        </ul>
-        <p>Improvements:</p>
-        <ul>
-          <li>German hyphenation patterns were added</li>
-        </ul>
-        <p>Translations:</p>
-        <ul>
-          <li>Dutch translation updated by Wilbert Berendsen</li>
-          <li>Italian translation updated by Davide Liessi</li>
-        </ul>
-      </description>
-    </release>
-    <release version="3.1.1" date="2020-01-04">
-      <description>
-        <p>Bugfixes:</p>
-        <ul>
-          <li>Fixed the error AttributeError: 'PopplerDocument' has no attribute 'ispresent'.</li>
-          <li>Fixed the error NameError: 'QPinchGesture' is not defined (pinch in music view).</li>
-          <li>Fixed the error NameError: 'doc' is not defined (when printing on Mac OSX).</li>
-        </ul>
-      </description>
-    </release>
-    <release version="3.1.0" date="2019-12-27">
-      <description>
-        <p>Requirement changes:</p>
-        <ul>
-          <li>Frescobaldi now requires Python3.3+</li>
-        </ul>
-        <p>New features:</p>
-        <ul>
-          <li>New "Document Fonts" dialog supporting text and music fonts and providing a font sample previewer.</li>
-          <li>Possibility to load external extensions.</li>
-          <li>New "First System Only" option in Custom Engrave.</li>
-          <li>New "Goto Line" and "Rename file" commands.</li>
-          <li>Music View: view settings are remembered per-document.</li>
-          <li>Music View: copy to Image can copy/export to SVG, PDF and EPS in addition to PNG/JPG.</li>
-          <li>Music View: new commands to rotate left/right; new toolbar button to show/hide the magnifier.</li>
-          <li>Music View: new preferences Horizontal/Vertical and Continuous/non-continuous.</li>
-          <li>Music View: new raster layout mode (displays as many pages as fit in a View).</li>
-          <li>Manuscript viewer: new toolbar buttons to rotate the pages left/right and to show/hide the magnifier.</li>
-        </ul>
-        <p>Bug fixes:</p>
-        <ul>
-          <li>Fixed seeking in MIDI player during playing stops sound.</li>
-          <li>Paper orientation is now properly handled in New Score Wizard.</li>
-          <li>Fixed discrepancy of LilyPond vs. system version of GhostScript
-    on Linux.</li>
-          <li>Fixed includepaths on Windows.</li>
-          <li>Fixed the error NameError: name 'widgets' is not defined.</li>
-        </ul>
-        <p>Improvements:</p>
-        <ul>
-          <li>Score Wizard: Titles and Headers are now shown in preview; new checkbox to write/omit pitches after \relative command; allow "none" for instrument names on first system.</li>
-          <li>Smarter behaviour of the autocompletion popup (#918, #922).</li>
-          <li>New command File->Rename/Move.</li>
-          <li>Sessions can be grouped in the Sessions menu.</li>
-          <li>Show absolute path of include files in tooltip.</li>
-          <li>Restructure Tools Menu.</li>
-          <li>File Open toolbar button shows recent files menu on long click.</li>
-          <li>Added "Blank Sheet Music" template snippet.</li>
-        </ul>
-      </description>
-    </release>
-    <release version="3.0.0" date="2017-02-17">
-      <description>
-        <p>Requirement changes:</p>
-        <ul>
-          <li>Frescobaldi now requires Python3.2+, Qt5, PyQt5, python-poppler-qt5.</li>
-        </ul>
-        <p>New features:</p>
-        <ul>
-          <li>Zoom with pinch gesture in Music View.</li>
-          <li>An option (enabled by default) to move the cursor to the end of the line when PageDown is pressed on the last line, and to move the cursor to the start of the first line if PageUp is pressed on the first line.</li>
-        </ul>
-        <p>Improvements:</p>
-        <ul>
-          <li>Retina display support in Music View.</li>
-        </ul>
-      </description>
-    </release>
+    <release version="3.2"   date="2022-05-05" />
+    <release version="3.1.3" date="2020-12-26" />
+    <release version="3.1.2" date="2020-04-13" />
+    <release version="3.1.1" date="2020-01-04" />
+    <release version="3.1.0" date="2019-12-27" />
+    <release version="3.0.0" date="2017-02-17" />
   </releases>
 
   <content_rating type="oars-1.1" />
