diff --git a/libguile/load.c b/libguile/load.c
index c209812dc..05f47c09e 100644
--- a/libguile/load.c
+++ b/libguile/load.c
@@ -550,6 +550,7 @@ static int
 compiled_is_fresh (SCM full_filename, SCM compiled_filename,
                    struct stat *stat_source, struct stat *stat_compiled)
 {
+  return 1;
   int compiled_is_newer;
   struct timespec source_mtime, compiled_mtime;
 
@@ -569,8 +570,6 @@ compiled_is_fresh (SCM full_filename, SCM compiled_filename,
       scm_display (compiled_filename, scm_current_warning_port ());
       scm_puts ("\n", scm_current_warning_port ());
     }
-
-  return compiled_is_newer;
 }
 
 static SCM

